genie-web/src/main/java/com/netflix/genie/web/data/repositories/jpa/specifications/JpaCommandSpecs.java

```diff
@@ -125,4 +130,40 @@ private JpaCommandSpecs() {
             return cb.and(predicates.toArray(new Predicate[0]));
         };
     }
+
+    /**
+     * Get the specification for the query which will find the commands which match the given criterion.
```

Comment:
Couple of things that would be nice to comment:
 - Implicitly looks for active
 - Cannot get a query that looks for commands in any state using this (i.e. all commands with tag `foo` regardless of state
 - An idea of the resulting SQL, especially the join/count part 

Just for future readability/understandability.